src/audio/CMakeLists.txt

```diff
@@ -175,9 +175,11 @@ set(sof_audio_modules mixer volume src asrc eq-fir eq-iir dcblock crossover tdfb
 if(CONFIG_IPC_MAJOR_3)
```

Comment:
Typo in commit message "mvoe". Also start sentences with capital letter.

Reply:
You are missing capital letter sentence starts from other commits as well,  this time ok for me but try to remember to write well the commit messages.